day_7/checker_board.rb

```diff
@@ -0,0 +1,39 @@
+# Output =>
+# X X X
+#  X X X
+# X X X
+#  X X X
+# X X X
+#  X X X
+
+
+p "How many lines of the Checker Board would you like to make?"
+
+def checkerboard()
+
+lines = gets.chomp
+checker_board = "X" + " " + "X" + " " + "X"
+line_2 = " " + "X" + " " + "X" + " " + "X"
+
+if lines.to_i.odd? && lines.to_i > 0
+  until lines.to_i == 1
+    p checker_board
+    lines = lines.to_i - 1
+    if lines.to_i.even?
+    p line_2
+    end
+
+  lines = lines.to_i - 1
+
+  end
+
+elsif lines.to_i.even?
+  until lines.to_i == 0
+  p "X" + " " + "X" + " " + "X"
+  p " " + "X" + " " + "X" + " " + "X"
+  lines = lines.to_i - 2
+  end
+end
+end
+
+p checkerboard
```

Comment:
Nice start here! I am noticing when I input an even number, I get the correct amount of rows, then a nil output, whereas when I try an odd number, I get one less row, but also a nil output, so there may be some logic to more closely examine. Also, a next step would be to make how long your rows are dynamic. Like when a user inputs 9, their rows will be "X X X X X".